include/luabind/bind.hpp

```diff
@@ -84,6 +84,26 @@ class class_ {
         return *this;
     }
 
+    template <auto func>
+    class_& static_function(const char* name) {
```

Comment:
Lets rename this to class_function ... It is more correct terminology generic language wise.

Reply:
done